expense/starter/main.go

```diff
@@ -25,8 +28,9 @@ func main() {
 		ID:        "expense_" + expenseID,
 		TaskQueue: "expense",
 	}
+	companyID := r.Intn(3)
```

Comment:
Company ID is not known ahead of time, so I can't start the workflow when the worker starts.